src/Microsoft.AspNetCore.Server.HttpSys/HttpSysOptions.cs

```diff
@@ -137,6 +140,38 @@ public long RequestQueueLimit
         /// </summary>
         public bool AllowSynchronousIO { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets a value that controls how http.sys reacts when rejecting requests due to throttling conditions - like when the request
+        /// queue limit is reached. The default in http.sys is "Basic" which means http.sys is just resetting the TCP connection. IIS uses Limited
+        /// as its default behavior which will result in sending back a 503 - Service Unavailable back to the client.
+        /// </summary>
+        public Http503ResponseVerbosityLevel Http503ResponseVerbosityLevel
```

Comment:
Does the enum name and property need to be this verbose? 😃 
How about Http503VerbosityLevel for the enum and Http503Verbosity for the property?

Reply:
Resolved in next iteration